src/schema/examples/msIdentifier.xml

```diff
@@ -2,6 +2,10 @@
 <egXML xmlns="http://www.tei-c.org/ns/Examples">
   <msIdentifier>
     <idno source="https://suche.staatsarchiv.djiktzh.ch/detail.aspx?ID=3737536" xml:lang="de">StAZH A III 2, S. 353-355</idno>
-    <repository xml:lang="de">Staatsarchiv Zürich</repository>
+    <repository xml:lang="de">Staatsarchiv</repository>
+    <settlement>Zürich</settlement>
```

Comment:
```suggestion
    <repository xml:lang="de">Staatsarchiv Zürich</repository>
    <settlement ref="loc000066">Zürich</settlement>
```

Reply:
Settlement anpasst in [a14bf90](https://github.com/SSRQ-SDS-FDS/ssrq-schema/pull/144/commits/a14bf90c8c604e901422f8eaf063084fdaf7ad9d)